src/nidmm/metadata/functions.py

```diff
@@ -1997,6 +2042,8 @@
                 'documentation': {
                     'description': 'Pass the value that you want to set the attribute to.'
                 },
+                'grpc_enum': 'NiDmmReal64AttributeValues',
+                'grpc_mapped_enum': 'NiDmmReal64AttributeValuesMapped',
```

Comment:
@reckenro Why did we add both a 'grpc_enum' and a 'grpc_mapped_enum' here? Is this correct??

Reply:
Yes, well at least that aligns with what's in grpc-device. You can see that this param has both the `attribute_value` for the enum and `attribute_value_mapped` for the mapped_enum [here in the .proto](https://github.com/ni/grpc-device/blob/main/generated/nidmm/nidmm.proto#L1444).

For the historical context, my memory's hazy but I think the "regular" enums are where the underlying value can be represented directly in the .proto enum value (i.e., in a 32-bit int). Otherwise, it gets put in a *Mapped enum.

I don't know if that was necessarily the right way to do things but that's what that is. So in grpc-device most often you'll just see `'enum'` keys because the corresponding values can be represented naturally in the enum values but sometimes you'll see a `'mapped-enum'` key and that's what that signifies.